pep-0692.rst

```diff
@@ -129,12 +136,12 @@ sections relates to type checker errors.
 Function calls with standard dictionaries
 -----------------------------------------
 
-Calling a function that has ``**kwargs`` typed using the ``**kwargs: **Movie``
-syntax with a dictionary of type ``dict[str, object]`` must generate a type
-checker error. On the other hand, the behaviour for functions using standard,
-untyped dictionaries can depend on the type checker. For example::
+Passing a dictionary of type ``dict[str, object]`` as a ``**kwargs`` argument
+to a function that has them typed using ``Unpack`` must generate a type checker
```

Comment:
"them" is unclear to me here, please could you reword?
```suggestion
to a function that has them typed using ``Unpack`` must generate a type checker
```

Reply:
Yeah, I think they might mean this?

```suggestion
to a function that has ``**kwargs`` typed using ``Unpack`` must generate a type checker
```

Reply:
Reworded to `....that has ``**kwargs`` annotated with ``Unpack`` must generate...`